packages/itwinui-css/src/utils/icon/icon.scss

```diff
@@ -18,14 +18,31 @@ $iui-icon-sizes: (
   3xl: var(--iui-size-2xl)
```

Comment:
I tried removing `2xl` but it is used by avatar and progress indicator radial.
I tried removing `3xl` but it is used by avatar, file upload, and tile.